### ISSUE TITLE
can't read this card AID : A0000000030000 on ios

### ISSUE THREAD

**Issue by sabardotnet** [nfcim/flutter_nfc_kit#45]
in ios i can't read this tag card but working on android

{"type":"iso7816","standard":"ISO 14443-4 (Type A)","id":"B552C648","atqa":"0400","sak":"20","historicalBytes":"80318066B1840C016E0183009000","hiLayerResponse":"","protocolInfo":"","applicationData":"","manufacturer":"","systemCode":"","dsfId":"","ndefAvailable":false,"ndefType":"","ndefCapacity":0,"ndefWritable":false,"ndefCanMakeReadOnly":false}

AID : A000000003000000

**Comment by jiegec** [comment]
@sabardotnet Please add the aid to the allowed aid list of plots. If that still doesn’t work, it might be forbidden by Apple, for example bank cards are not readable.

**Comment by jiegec** [comment]
@jiegec Plots -> Plists

**Comment by Harry-Chen** [comment]
Seems to duplicate with #46.

**Comment by bradgrissom** [comment]
I had the same issue with the demo/example app for flutter_nfc_kit and I found that I needed to use a more generic "NDEF" AID (Application ID) of the following:

D2 76 00 00 85 01 01

I had to put that AID in the Info.plist as well as verify that my NFC tag would respond to that AID.  In my case I'm using a NFC Tag Emulator from ST Micro, so I have full control over how it behaves.  By doing the above, everything is working properly and I can read and write NDEF information on my emulated tag.

As the original poster mentioned, responding to:
AID : A000000003000000

Gave me some sort of iOS invalid tag error.  As mentioned in this thread, this may be a smartcard AID (EMVCo?) which may or may not be specific to ST Micro.  My guess is that to make that work properly, you'd need extra iOS/Apple entitlements in your app for contactless payment (EMVCo).
